### PR TITLE
Make coveralls watch the include folder

### PR DESCRIPTION
**Pull request by theodelrieu**
Should be enough to fix it.

### CODE REVIEW

**Comment by theodelrieu** [comment]
The latest Coveralls job passed: https://coveralls.io/jobs/33439123

Could you retry the Travis job that failed? Seems like a spurious failure...

**Comment by theodelrieu** [comment]
Oops, seems my fix didn't fix anything after all...

**Comment by nlohmann** [comment]
coveralls

This is how [Coveralls](https://coveralls.io/builds/15376538) looks like right now. The 0% at the top is strange - and this is also the number reported in this PR. Any idea how this can be fixed?

**Comment by theodelrieu** [comment]
That's very strange, maybe because we give `include` instead of `include/nlohmann`, I'll give it a try.

**Comment by theodelrieu** [comment]
I might become crazy with Coveralls... @nlohmann I remember I could try locally before, but the private repo's token has been moved from the repo files (which is good!)

Could you try to fix this one? Running Travis for hours to finally see a big fat **0.0%** is quite frustrating...

**Comment by nlohmann** [comment]
I'll have a look.

**Comment by nlohmann** [comment]
See comments in #953. With 2dda87c3b7440ceac5b7ffa93de5c9bc1e85a57f, coverage data should be generated again. I am currently not planning to extend this to the multi-header version myself, but I think this can be done with a PR, because Travis/Coveralls seems not to care about who is comitting.

**Comment by theodelrieu** [comment]
It seems CMake does not go well with environment variables which contains spaces...

Hence the `MULTIPLE_HEADERS` I've added in the travis file

**Comment by theodelrieu** [comment]
I think it's good now, finally!

File: .travis.yml [hunk 1]
```diff
@@ -281,14 +282,16 @@ script:
 
   # make sure CXX is correctly set
   - if [[ "${COMPILER}" != "" ]]; then export CXX=${COMPILER}; fi
+  # by default, use the single-header version
+  - if [[ "${MULTIPLE_HEADERS}" == "" ]]; then export MULTIPLE_HEADERS=OFF; fi
```
Comment (nlohmann) [inline .travis.yml]: I did not think of this trick. Well done!

**Pull request merged**